src/mono/mono/mini/mini.c

```diff
@@ -3758,6 +3758,7 @@ mini_method_compile (MonoMethod *method, guint32 opts, JitFlags flags, int parts
 		cfg->init_method_rgctx_ins = NULL;
 		cfg->init_method_rgctx_ins_arg->opcode = OP_PCONST;
 		cfg->init_method_rgctx_ins_arg->inst_p0 = NULL;
+		MONO_INST_NULLIFY_SREGS (cfg->init_method_rgctx_ins_arg);
```

Comment:
is it surprising that init_method_rgctx_ins_arg has non-zero sregs? It seems like it should always be an OP_AOTCONST or an OP_PCONST

Reply:
oh, I see. it's `OP_GOT_ENTRY` that has a `(dest)->inst_basereg = __got_loc->dreg;` which is `sreg1`